fedgov-cv-message-builder/src/main/java/gov/usdot/cv/msg/builder/input/IntersectionInputData.java

```diff
@@ -275,7 +275,7 @@ public static class ReferencePointChild {
 		public RegulatorySpeedLimit[] speedLimitType;	
 	}
 	
-	// @JsonTypeName("speedLimitType")
+	// @JsonTypeName("speedLimitType") 

```

Comment:
NIT: In the future please remove any commented out code that is no longer being used. Leaving commented out sections in our code makes it harder to read especially when there is not comment for context.